### PR TITLE
Add section point to microk8s

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

File: docs/source/admin/index.rst [hunk 1]
```diff
@@ -16,12 +16,14 @@ For medium-sized deployments (a handful of users), an AiiDAlab multi-user server
 Since deploying a multi-user server requires additional packages, such as `JupyterHub <https://jupyter.org/hub>`__, `DockerSpawner <https://github.com/jupyterhub/dockerspawner>`__, `Apache HTTP Server <https://www.apache.org/>`__, and `Docker <http://www.docker.com>`__, we provide the ``aiidalab-server`` `Ansible <https://www.ansible.com/>`__ role which automates the setup of the server.
 Please see the `corresponding git repository <https://github.com/aiidalab/ansible-role-aiidalab-server>`__ for more information.
 
+We also provide `instructions for deploy with microk8s in a single-server setup <https://github.com/aiidalab/aiidalab-microk8s-deploy>`__. This is a good option if you want to deploy AiiDAlab on a single machine, but expect a large number of users (>50).
```
Comment (danielhollas) [inline docs/source/admin/index.rst:19]: Oh, I did not realize this exists! Looks like a perfect solution for me as well in the future! 
Two minor suggestions:
1. Maybe link directly to README?
2. Based on the README, it looks like this is for groups of <50 people, not >50 people?

```suggestion
We also provide `instructions for deploy with microk8s in a single-server setup <https://github.com/aiidalab/aiidalab-microk8s-deploy#readme>`. This is a good option if you want to deploy AiiDAlab on a single machine, but expect a moderate amount of users (<50).
```

Comment (unkcpz) [reply docs/source/admin/index.rst:19]: Ah, yes! Damn copilot :smile:

File: docs/source/admin/index.rst [hunk 2]
```diff
@@ -16,12 +16,14 @@ For medium-sized deployments (a handful of users), an AiiDAlab multi-user server
 Since deploying a multi-user server requires additional packages, such as `JupyterHub <https://jupyter.org/hub>`__, `DockerSpawner <https://github.com/jupyterhub/dockerspawner>`__, `Apache HTTP Server <https://www.apache.org/>`__, and `Docker <http://www.docker.com>`__, we provide the ``aiidalab-server`` `Ansible <https://www.ansible.com/>`__ role which automates the setup of the server.
 Please see the `corresponding git repository <https://github.com/aiidalab/ansible-role-aiidalab-server>`__ for more information.
 
+We also provide `instructions for deploy with microk8s in a single-server setup <https://github.com/aiidalab/aiidalab-microk8s-deploy#readme>`. This is a good option if you want to deploy AiiDAlab on a single machine, but expect a moderate amount of users (<50).
```
Comment (danielhollas) [inline docs/source/admin/index.rst:19]: I think there is a mistake here, if you checkout the rendering at https://aiidalab--393.org.readthedocs.build/en/393/admin/index.html#single-server-deployment (introduced by my suggestion, I did not realize this is a rst file and not markdown).

```suggestion
We also provide `instructions for deploy with microk8s in a single-server setup <https://github.com/aiidalab/aiidalab-microk8s-deploy#readme>`__. This is a good option if you want to deploy AiiDAlab on a single machine, but expect a moderate amount of users (<50).
```

**Review by danielhollas — APPROVED** [review]
Looks good apart from one rendering issue

**Review by yakutovicha — APPROVED** [review]

**Pull request merged**